install

```diff
@@ -123,16 +123,21 @@ if ! command_exists tar; then
 fi
 
 if [ "x${MESHERY_VERSION}" = "x" ] ; then
-  MESHERY_VERSION=$(curl -L -s https://github.com/meshery/meshery/releases/latest | \
-                  grep "meshery/releases/tag/" | head -1 | awk -F '"' '{print $4}' | \
-                  awk -F '/' '{print $NF}' )
+  MESHERY_VERSION=$(curl -L -s https://docs.meshery.io/project/releases/latest)
 fi
 
 if [ "x${MESHERY_VERSION}" = "x" ] ; then
   printf "Unable to get latest mesheryctl version. Set MESHERY_VERSION env var and re-run. For example: export MESHERY_VERSION=v0.1.6\n"
   exit;
 fi
 
+# For patch releases, the meshery tar ball will not be generated, so fall back to last stable release
```

Comment:
I see other type of releases as well, e.g. `beta`, `rc`. Do we not need special handling for those cases?

Reply:
@nishantwrp, for beta and rc releases, tar balls are available.
For e.g. 
https://github.com/meshery/meshery/releases/tag/v0.7.0-beta-4
https://github.com/meshery/meshery/releases/tag/v0.6.0-rc-2
https://github.com/meshery/meshery/releases/tag/v0.6.0-rc-5j